### PR TITLE
added example for blocking device classes

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6658

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @mypil, for your information

Thank you

File: windows/security/threat-protection/device-control/control-usb-devices-using-intune.md [hunk 1]
```diff
@@ -145,6 +145,14 @@ Get-WMIObject -Class Win32_DiskDrive |
 Select-Object -Property * 
 ```
 
+The **Prevent installation of devices using drivers that match these device setup classes** policy allows you to specify device setup classes that Windows is prevented from installing. 
+
+To prevent installation of particular classes of devices: 
+
+1. Find the GUID of the device setup class from [System-Defined Device Setup Classes Available to Vendors](https://docs.microsoft.com/windows-hardware/drivers/install/system-defined-device-setup-classes-available-to-vendors)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/device-control/control-usb-devices-using-intune.md:152]:
```suggestion
1. Find the GUID of the device setup class from [System-Defined Device Setup Classes Available to Vendors](https://docs.microsoft.com/windows-hardware/drivers/install/system-defined-device-setup-classes-available-to-vendors).
```

**Comment by MaratMussabekov** [comment]
Hello, could you please check image again?
I see it following way:
image
I use this link: https://github.com/MaratMussabekov/windows-itpro-docs/blob/patch-158/windows/security/threat-protection/device-control/control-usb-devices-using-intune.md
Thank you

**Comment by JohanFreelancer9** [comment]
@MaratMussabekov The address that is linked appears to be https://github.com/MicrosoftDocs/windows-itpro-docs/raw/public/windows/security/threat-protection/device-control/images/Add-device-setup-class-to-prevent-list.png, whereas it should be https://github.com/MaratMussabekov/windows-itpro-docs/blob/patch-158/windows/security/threat-protection/device-control/images/admintemplates.png. Perhaps I am wrong but could you check again, please.

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #6658.
 
Thank you.

cc: @kenwith

**Pull request merged**